geopandas/tests/test_merge.py

```diff
@@ -63,3 +72,23 @@ def test_concat_axis1(self):
         assert isinstance(res, GeoDataFrame)
         assert isinstance(res.geometry, GeoSeries)
         self._check_metadata(res)
+
```

Comment:
```suggestion

    def test_concat_axis1_multiple_geodataframes(self):
```

(to have it as a separate test, otherwise the single test is becoming long)